containers/src/impl/Kokkos_UnorderedMap_impl.hpp

```diff
@@ -47,16 +47,15 @@ struct UnorderedMapRehash {
   using const_map_type  = typename map_type::const_map_type;
   using execution_space = typename map_type::execution_space;
   using size_type       = typename map_type::size_type;
+  using policy_type =
+      Kokkos::RangePolicy<execution_space, IndexType<size_type>>;
 
   map_type m_dst;
   const_map_type m_src;
 
-  UnorderedMapRehash(map_type const& dst, const_map_type const& src)
-      : m_dst(dst), m_src(src) {}
-
-  void apply() const {
-    parallel_for("Kokkos::Impl::UnorderedMapRehash::apply", m_src.capacity(),
-                 *this);
+  void apply(const execution_space& space = execution_space{}) const {
```

Comment:
Is there a reason to keep the default argument? Since this is not user-facing, I would rather add passing an execution space instance on the call site.

Reply:
I agree, I removed the default.